lib/lexisnexis/request.rb

```diff
@@ -7,10 +7,11 @@ module LexisNexis
   class RequestError < StandardError; end
 
   class Request
-    attr_reader :attributes, :url, :headers, :body
+    attr_reader :config, :applicant, :url, :headers, :body
 
-    def initialize(attributes)
-      @attributes = attributes
+    def initialize(config:, applicant:)
+      @config = config
+      @applicant = applicant
```

Comment:
This includes a rename from `attributes` to `applicant` because the top-level `Proofer` interface uses `applicant` and I decided things might be easier if everything used the same terminology (instead of having a `attributes: applicant` in there somewhere -- this results in some diff noise throughout the PR